src/Magnum/ImGuiIntegration/Test/ContextGLTest.cpp

```diff
@@ -742,6 +809,282 @@ void ContextGLTest::multipleContexts() {
     CORRADE_COMPARE(ImGui::GetCurrentContext(), a.context());
 }
 
+constexpr Color4 DrawClearColor{0.5f, 0.5f, 1.0f, 1.0f};
+
+void ContextGLTest::drawSetup() {
+    GL::Renderer::setClearColor(DrawClearColor);
+
+    constexpr Vector2i DrawSize{100, 100};
+
+    _color = GL::Renderbuffer{};
+    _color.setStorage(
+        #if !defined(MAGNUM_TARGET_GLES2) || !defined(MAGNUM_TARGET_WEBGL)
+        GL::RenderbufferFormat::RGBA8,
+        #else
+        GL::RenderbufferFormat::RGBA4,
+        #endif
+        DrawSize);
+
+    _framebuffer = GL::Framebuffer{{{}, DrawSize}};
+    _framebuffer
+        .attachRenderbuffer(GL::Framebuffer::ColorAttachment{0}, _color)
+        .clear(GL::FramebufferClear::Color)
+        .bind();
+}
+
+void ContextGLTest::drawTeardown() {
+    _framebuffer = GL::Framebuffer{NoCreate};
+    _color = GL::Renderbuffer{NoCreate};
+}
+
+void ContextGLTest::draw() {
+    Context c{{200, 200}, {70, 70}, _framebuffer.viewport().size()};
+
+    /* ImGui doesn't draw anything the first frame */
+    c.newFrame();
+    c.drawFrame();
+
+    MAGNUM_VERIFY_NO_GL_ERROR();
+
+    Utility::System::sleep(1);
+
+    c.newFrame();
+
+    /* Last drawlist that gets rendered, covers the entire display */
+    ImDrawList* drawList = ImGui::GetForegroundDrawList();
+    const ImVec2& size = ImGui::GetIO().DisplaySize;
+
+    drawList->AddRectFilled({size.x*0.1f, size.y*0.2f}, {size.x*0.9f, size.y*0.8f},
+        IM_COL32(255, 128, 128, 255));
+    drawList->AddRectFilled({size.x*0.5f, size.y*0.5f}, size,
+        /* Transparent to make sure alpha blending works (blend state and draw
+           order shouldn't be changed by drawFrame()) */
+        IM_COL32(255, 255, 255, 128));
+    drawList->AddTriangleFilled({0.0f, 0.0f}, {size.x*0.5f, 0.0f}, {size.x*0.25f, size.y*0.5f},
+        IM_COL32(128, 255, 128, 255));
+
+    c.drawFrame();
+
+    MAGNUM_VERIFY_NO_GL_ERROR();
+
+    if(!(_manager.loadState("AnyImageImporter") & PluginManager::LoadState::Loaded) ||
+       !(_manager.loadState("TgaImporter") & PluginManager::LoadState::Loaded))
+        CORRADE_SKIP("AnyImageImporter / TgaImporter plugins not found.");
```

Comment:
Consistently with how it's handled e.g. in case of `KtxImporter` in `AnyImageImporter` tests as of mosra/magnum@578baf3168b50da73d692d48ffa878c19e313baa -- using `load()` will catch not only absence of the plugin, but also a case when it doesn't link because of an ABI change:

```suggestion
    /* Catch also ABI and interface mismatch errors */
    if(!(manager.load("AnyImageImporter") & PluginManager::LoadState::Loaded) ||
       !(manager.load("PngImporter") & PluginManager::LoadState::Loaded))
        CORRADE_SKIP("AnyImageImporter / PngImporter plugin can't be loaded.");
```

Similarly in the other case below.

(I already switched to PNGs here, yes.)